packages/demo/src/configuration/blocks/effects/blurBlock.serializer.ts

```diff
@@ -0,0 +1,29 @@
+import type { BaseBlock, IBlockSerializerV1 } from "@babylonjs/smart-filters";
+import type { BlurBlock } from "./blurBlock";
+import { BlockNames } from "../blockNames";
+
+/**
+ * The V1 serializer for a Blur Block.
+ * Though it is an aggregate block, Blur creates and manages its own blocks
+ * internally, so there's no need to worry about serializing them.
+ */
+export const blurBlockSerializer: IBlockSerializerV1 = {
+    className: BlockNames.blur,
+    serialize: (block: BaseBlock) => {
```

Comment:
I gave this a go, but ran into problems because I couldn't figure out how to accommodate the specific IBlockSerializerV1 types in containers like `additionalBlockSerializers` without doing something like
```typescript
const additionalBlockSerializers: IBlockSerializerV1<any>[] = {
    blurBlockSerializer, // type is IBlockSerializerV1<BlurBlock>,
    ...
}
```
or
```typescript
const additionalBlockSerializers: IBlockSerializerV1<BaseBlock>[] = {
    blurBlockSerializer as IBlockSerializerV1<BaseBlock>,
    ...
}
```
which feels... wrong. Will come back to this later